catalogs/catalog-jdbc-postgresql/src/main/java/com/datastrato/gravitino/catalog/postgresql/operation/PostgreSqlTableOperations.java

```diff
@@ -42,8 +42,10 @@ public class PostgreSqlTableOperations extends JdbcTableOperations {
   public static final String PG_QUOTE = "\"";
   public static final String NEW_LINE = "\n";
   public static final String ALTER_TABLE = "ALTER TABLE ";
-  public static final String ALTER_COLUMN = " ALTER COLUMN ";
+  public static final String ALTER_COLUMN = "ALTER COLUMN ";
   public static final String IS = " IS '";
```

Comment:
Could you give this variable a better name?

Reply:
Like what? That name is in the same form as "ALTER_TABLE " I'm not sure you would want it different to that.

Reply:
This string contains `'`. So I wonder whether its name should tell user.

Reply:
I didn't change or write that line. My guess is because no code currently used IS without quotes, it makes sense.

Reply:
@Clearvive
Does the first blank character in "ALTER_COLUMN" serve a specific purpose?

Reply:
It did, but I refactored the code so " " was used instead where needed.